packages/beta/src/api/documents/previewApi.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Cognite AS
+
+import { BaseResourceAPI } from '@cognite/sdk-core';
+
+import { DocumentId, DocumentPreviewTemporaryLink } from '../../types';
+import Any = jasmine.Any;
+
+export class PreviewAPI extends BaseResourceAPI<Any> {
+  public document = (
```

Comment:
How exactly can I go about supporting file download?

and also how can I remove the "accept: application/json" header that is being injected?